### ISSUE TITLE
Opis not finding MobiusCoreTH

### ISSUE THREAD

**Issue by Penrouk** [CyberdyneCC/MobiusCoreTH#1]
I'm running a Thermos server and wanted to get Opis on to it. After awhile of searching around, I found this and got MobiusCoreTH working for the server. The issue is Opis 1.2.5 cannot find the MobiusCore it needs to run resulting in a crash. I have attached the crash report for this issue.
[crash-2016-07-27_02.40.56-server.txt](https://github.com/CyberdyneCC/MobiusCoreTH/files/385555/crash-2016-07-27_02.40.56-server.txt)

**Comment by ArcasCZ** [comment]
MobiusCoreTH is not properly installed. Are you using right file? Can you send me that file?

**Comment by Penrouk** [comment]
Sure, I'm not good with using sources. What I did was put the stuff into a current 1.2.5 file and replaced the old stuff. I attached the file. For some reason it wouldn't let me attach file so I made a dropbox link for it. https://dl.dropbox.com/s/exrjem61td4wmtm/MobiusCore-1.2.5_1.7.10.jar?dl=0
Knowing me though, I probably messed it up. 
Here is also to Opis I'm using: https://dl.dropbox.com/s/hatl8nwgxr6c3jo/Opis-1.2.5_1.7.10.jar?dl=0
